service.py

```diff
@@ -0,0 +1,60 @@
+from concurrent import futures
+import grpc
+import time
+import utility
+import hwsc_file_transaction_svc_pb2
+import hwsc_file_transaction_svc_pb2_grpc
+
+class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+     def __init__(self):
+
+         class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+             def __init__(self):
+                 pass
+
+             def GetStatus(self, request, context):
+                 print("Get Status")
+
+             def DownloadZippedFiles(self, request_iterator, context):
+                 if request_iterator.name:
+                     return utility.download_chunk(self.tmp_file_name)
+
+             def UploadFile(self, request_iterator, context):
+                 print("[INFO] Requesting UploadFile service")
+
+                 for get_name in request_iterator:
+                     get_url = utility.upload_file_to_azure(request_iterator, get_name.file_name)

```

Comment:
This line can throw `NoSuchBlobException`. We should handle it using a `try-catch` block,

Reply:
The `catch` block is a good place to return an error.

Reply:
Add `# TODO` if you will fix this for the next PR.